streams/src/main/java/org/apache/kafka/streams/kstream/Branched.java

```diff
@@ -125,21 +149,14 @@ public static <K, V> Branched<K, V> withConsumer(final Consumer<? super KStream<
         return new Branched<>(name, null, chain);
     }
 
-    /**
-     * Create an instance of {@code Branched} from an existing instance.
-     *
-     * @param branched the instance of {@code Branched} to copy
-     */
-    protected Branched(final Branched<K, V> branched) {
```

Comment:
I'm assuming this is just for moving the func around, and removing the javadoc for those protected functions?

Reply:
Yes.